test/over_react/util/map_util_test.dart

```diff
@@ -20,13 +20,12 @@ import 'package:test/test.dart';
 import 'package:over_react/src/component_declaration/component_base.dart' as component_base;
 
 typedef ForwardUnconsumedPropsFunction = void Function(Map props, {
-    bool omitReactProps,
-    bool onlyCopyDomProps,
-    Iterable keysToOmit,
-    Iterable<Iterable>
-    keySetsToOmit,
-    Map propsToUpdate
-  });
+  bool omitReactProps,
+  bool onlyCopyDomProps,
```

Comment:
It looks like the functions being assigned to this typedef type are optional non-nullable (they have defaults), so no. It 
 does look a little weird in this typedef since they don't have defaults